the-blue-alliance-ios/View Controllers/Settings/SettingsTableViewController.swift

```diff
@@ -0,0 +1,200 @@
+        self.dismiss(animated: true, completion: nil)
+    }
+    
+    // MARK: - Table view configuration
+    
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return 4
+    }
+    
+    override func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        switch section {
+        case 0:
+            return "Account"
+            
+        case 1:
+            return "Notifications"
+            
+        case 2:
+            return "Info"
+            
+        case 3:
+            return "Extra"
+            
+        default:
+            fatalError("This section does not exist")
+        }
+    }
+    
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        switch section {
+        case 0:
+            return 1
+            
+        case 1:
+            return 1
+            
+        case 2:
+            return 4
+            
+        case 3:
+            return 2
+            
+        default:
+            fatalError("This section does not exist")
+        }
+    }
+    
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "BaseCell", for: indexPath)
+        
+        let thirdSectionLabels =  ["The Blue Alliance Website",  "The Blue Alliance for iOS is open source", "Contributors", "View changelog"]
+        let lastSectionLabels = ["Delete network cache", "Delete app data"]
+        
+        switch indexPath.section {
+        case 0:
+            let cell = tableView.dequeueReusableCell(withIdentifier: "SwitchCell", for: indexPath) as! SwitchTableViewCell
+            cell.populate(withTitle: "Enable myTBA")
+            
+            // Configure the cell's switch state depending on wether the user is connected or not
+            if MyTBA.shared.authentication != nil {
+                cell.switchToggle.isOn = true
+            } else {
+                cell.switchToggle.isOn = false
+            }
+            
+            return cell
+            
+        case 1:
+            let cell = tableView.dequeueReusableCell(withIdentifier: "SwitchCell", for: indexPath) as! SwitchTableViewCell
+            cell.populate(withTitle: "Enable notifications")
+            
+            return cell
+
+        case 2:
+            cell.textLabel?.text = thirdSectionLabels[indexPath.row]
+            
+        case 3:
+            cell.textLabel?.text = lastSectionLabels[indexPath.row]
+            
+            cell.textLabel?.textColor = .red
+            cell.textLabel?.textAlignment = .center
+            
+        default:
+            fatalError("This row does not exist")
+        }
+        
+        return cell
+    }
+    
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        switch indexPath.section {
```

Comment:
Make sure we're deselecting these rows as well

Reply:
Actually, just move the deselect call to the top - we're calling it several places right now and we can just have one call it looks like